index.ts

```diff
@@ -640,6 +640,11 @@ export default class FreeSound {
     return this.makeRequest<AccessTokenResponse>(postUrl, 'POST', data);
   }
 
+  /**
+   * Search sounds in Freesound by matching their tags and other kinds of metadata. See
+   * https://freesound.org/docs/api/resources_apiv2.html#sound-text-search for more
+   * information.
```

Comment:
One thing I forgot to mention in the corresponding ticket's description was that this task also includes adding API examples. The examples should:

* Use md ([typedoc suppports md](https://typedoc.org/guides/doccomments/#markdown)), see [code doc examples](https://typedoc.org/guides/doccomments/#code-blocks)
* The examples should be realistic:

**Bad** (no syntax highlighting, usage of variable names)

```
await freeSound.textSearch(bar, {
  page: foo,
  filter: someFillter
  sort: myDesc
  fields: someField
});
```

**Good**:  (syntax highlighting, usage of string literals for clear examples)
```ts
await freeSound.textSearch('violoncello', {
  page: 1,
  filter: 'tag:tenuto duration:[1.0 TO 15.0]',
  sort: 'rating_desc',
  fields: 'id,name,url'
});
```

Reply:
I attempted this with 131c1b0. I added API examples for methods that had examples in this repository, [Tide](https://github.com/tide-app/tide/search?q=setToken), or both.

Thank you for the Typedoc links and the "Good" and "Bad" API examples. I think those three things were very helpful.